.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I believe that this file + all files from the `dist` directory are committed by a mistake?

Reply:
Sorry, I removed the dist folder + added dist folders and .project in .gitignore file